common/modal.js

```diff
@@ -45,24 +45,24 @@
         vm.showReloadBtn = false;
         var reloadMessage = ' <p>  </p>';
 
-        if(vm.params.errorStatus == 'Multiple Records Found'){
+        if (vm.params.errorStatus == 'Multiple Records Found') {
```

Comment:
We can pass the actual error object to `errorPopup`, which then can be passed to the this controller. So we can use `instanceof` instead of checking for `errorStatus`.